export type StudioColors = 
	 "MainBackground"
	| "Titlebar"
	| "Dropdown"
	| "Tooltip"
	| "Notification"
	| "ScrollBar"
	| "ScrollBarBackground"
	| "TabBar"
	| "Tab"
	| "FilterButtonDefault"
	| "FilterButtonHover"
	| "FilterButtonChecked"
	| "FilterButtonAccent"
	| "FilterButtonBorder"
	| "FilterButtonBorderAlt"
	| "RibbonTab"
	| "RibbonTabTopBar"
	| "Button"
	| "MainButton"
	| "RibbonButton"
	| "ViewPortBackground"
	| "InputFieldBackground"
	| "Item"
	| "TableItem"
	| "CategoryItem"
	| "GameSettingsTableItem"
	| "GameSettingsTooltip"
	| "EmulatorBar"
	| "EmulatorDropDown"
	| "ColorPickerFrame"
	| "CurrentMarker"
	| "Border"
	| "DropShadow"
	| "Shadow"
	| "Light"
	| "Dark"
	| "Mid"
	| "MainText"
	| "SubText"
	| "TitlebarText"
	| "BrightText"
	| "DimmedText"
	| "LinkText"
	| "WarningText"
	| "ErrorText"
	| "InfoText"
	| "SensitiveText"
	| "ScriptSideWidget"
	| "ScriptBackground"
	| "ScriptText"
	| "ScriptSelectionText"
	| "ScriptSelectionBackground"
	| "ScriptFindSelectionBackground"
	| "ScriptMatchingWordSelectionBackground"
	| "ScriptOperator"
	| "ScriptNumber"
	| "ScriptString"
	| "ScriptComment"
	| "ScriptKeyword"
	| "ScriptBuiltInFunction"
	| "ScriptWarning"
	| "ScriptError"
	| "ScriptInformation"
	| "ScriptHint"
	| "ScriptWhitespace"
	| "ScriptRuler"
	| "DocViewCodeBackground"
	| "DebuggerCurrentLine"
	| "DebuggerErrorLine"
	| "DiffFilePathText"
	| "DiffTextHunkInfo"
	| "DiffTextNoChange"
	| "DiffTextAddition"
	| "DiffTextDeletion"
	| "DiffTextSeparatorBackground"
	| "DiffTextNoChangeBackground"
	| "DiffTextAdditionBackground"
	| "DiffTextDeletionBackground"
	| "DiffLineNum"
	| "DiffLineNumSeparatorBackground"
	| "DiffLineNumNoChangeBackground"
	| "DiffLineNumAdditionBackground"
	| "DiffLineNumDeletionBackground"
	| "DiffFilePathBackground"
	| "DiffFilePathBorder"
	| "ChatIncomingBgColor"
	| "ChatIncomingTextColor"
	| "ChatOutgoingBgColor"
	| "ChatOutgoingTextColor"
	| "ChatModeratedMessageColor"
	| "Separator"
	| "ButtonBorder"
	| "ButtonText"
	| "InputFieldBorder"
	| "CheckedFieldBackground"
	| "CheckedFieldBorder"
	| "CheckedFieldIndicator"
	| "HeaderSection"
	| "Midlight"
	| "StatusBar"
	| "DialogButton"
	| "DialogButtonText"
	| "DialogButtonBorder"
	| "DialogMainButton"
	| "DialogMainButtonText"
	| "InfoBarWarningBackground"
	| "InfoBarWarningText"
	| "ScriptEditorCurrentLine"
	| "ScriptMethod"
	| "ScriptProperty"
	| "ScriptNil"
	| "ScriptBool"
	| "ScriptFunction"
	| "ScriptLocal"
	| "ScriptSelf"
	| "ScriptLuauKeyword"
	| "ScriptFunctionName"
	| "ScriptTodo"
	| "ScriptBracket"
	| "AttributeCog"
	| "AICOOverlayText"
	| "AICOOverlayButtonBackground"
	| "AICOOverlayButtonBackgroundHover"
	| "AICOOverlayButtonBackgroundPressed"
	| "OnboardingCover"
	| "OnboardingHighlight"
	| "OnboardingShadow"
	| "BreakpointMarker"
	| "DiffLineNumHover"
	| "DiffLineNumSeparatorBackgroundHover"
export type StudioModifiers = 
	 "Default"
	| "Selected"
	| "Pressed"
	| "Disabled"
	| "Hover"

return {}